cmd/oj/main.go

```diff
@@ -304,6 +315,7 @@ func run() (err error) {
 		}
 	}
 	if len(files) == 0 && len(input) == 0 {
+		// TBD if dig ...
```

Comment:
Is this a TODO? Something forgotten?

Reply:
updated and removed comment, I missed that one.